.github/workflows/run_tests.yml

```diff
@@ -65,8 +65,8 @@ jobs:
       fail-fast: false
 
       matrix:
-        os: [ "macos-latest", "windows-latest", "ubuntu-20.04" ]
-        python-version: [ '3.8', '3.9', '3.10' ]
```

Comment:
Hrm. Strangely, the tests are hanging on [macOS + Python 3.12](https://github.com/ivadomed/ivadomed/actions/runs/15053047106/job/42369140854?pr=1319) even though _all the tests have passed_.

It looks like there might be some sort of issue on macOS with the cleanup steps for `pytest`... 🤔